rahul/exercise12 - Prompting people for numbers/prompting-people-for-numbers.rb

```diff
@@ -0,0 +1,12 @@
+print 'Give me a number: '
+number = gets.chomp.to_i
+
+bigger = number * 100
+puts "A bigger number is #{bigger}"
+
+print 'Give me another number: '
+another = gets.chomp
+number = another.to_i
+
+smaller = number / 100
+puts "A smaller number is #{smaller}."
```

Comment:
How about removing the extra variable here and at other similar places too: 

number = gets.chomp.to_i
puts "A smaller number is #{number / 100}."